bindings/csharp/http/batch/batch.csproj

```diff
@@ -2,13 +2,9 @@
 
     <PropertyGroup>
       <OutputType>Exe</OutputType>
-      <TargetFramework>net6.0</TargetFramework>
+      <TargetFramework>net8.0</TargetFramework>
       <Nullable>enable</Nullable>
       <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
-  
-    <ItemGroup>
-      <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
```

Comment:
Reviewed with .NET SDK maintainers in Dapr and they agree with this.  Also tests pass.